code/map.dm

```diff
@@ -30,6 +30,7 @@ var/global/list/mapNames = list(
 	"1 pamgoC" = 		list("id" = "PAMGOC", 		"settings" = "pamgoc", 			"playerPickable" = ASS_JAM),
 	"Kondaru" = 		list("id" = "KONDARU", 		"settings" = "kondaru", 		"playerPickable" = 1,		"MaxPlayersAllowed" = 80),
 	"Ozymandias" = 	list("id" = "OZYMANDIAS", "settings" = "ozymandias", 	"playerPickable" = 0, 	"MinPlayersAllowed" = 40),
+	"Nadir" = 		list("id" = "NADIR", 		"settings" = "nadir", 		"playerPickable" = 0),
```

Comment:
would not want admins to be able to pick it

Reply:
On the fence about that, honestly. While the map still needs some additional work to get it functioning precisely as intended, it's entirely playable and it would be useful to get some more information on how well not only the custom features but also the regular-station features operate.

That said: entirely alright with it being commented out if there's broad agreement that it should remain non-accessible until the feature set is built out.